### DISCUSSION TITLE
Disable Auto Download TV Show

### DISCUSSION THREAD

**Discussion by austinbarberrepo** [sct/overseerr#650] [Help] [answered]
Is there a way to disable the tv show from auto-downloading after being requested? I have seen this in the past with another app. A user will request a show and before I know it there is 100GBs in my downloader. I have it setup now to where I approve TV show requests. However, when I approve them it still starts the search for episodes. Am I missing a setting in Sonarr for this?

Thank you!

**Comment by sct** [comment]
There is no way to do this at the moment. We will be adding the ability to edit requests before you approve them, so you can limit some of the requested seasons. Would that help you?

**Reply by austinbarberrepo** [reply]
This can be closed. I saw that someone else opened an issue for this too. Issue 662. Thank you!

**Reply by austinbarberrepo** [reply]
I saw the new post for version 1.18. I see that my version has updated to it as it’s on the develop instance. I don’t see the checkbox tho. I tried to create a new sonarr instance to see if it would show but still doesn’t. Is there something extra I need to do? I’m running this on windows 10.

**Comment by austinbarberrepo** [comment]
Would that be the best way to reduce the number of seasons that it searches
for? When adding a new tv show to Sonarr there is a check box in the bottom
right that says "Start search for missing episodes''.  I'm not sure if that
is how it works on the backend. It seems like that box gets checked after a
show gets approved in Overseerr. Would it be possible to not have that box
checked after a show gets approved?

Thank you.

**Comment by samwiseg0** [comment, marked as answer]
This has been implemented with "Disable Auto-search" feature in both sonarr and radarr.
